org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/Util.java

```diff
@@ -828,4 +849,141 @@ public static int shiftmask(int bitwidth) {
         return bitwidth < 1 ? 0 : (1 << (32 - Integer.numberOfLeadingZeros(bitwidth - 1))) - 1;
     }
 
+
```

Comment:
```suggestion
```

Reply:
?

Reply:
Seems added newline is irrelevant change and could be removed.